lib/data/repositories/articles_directory_repository.dart

```diff
@@ -84,4 +85,10 @@ class ArticlesDirectoryRepository {
       List<String> articleReferences, int limit) async {
     return _firestoreManager.fetchArticlesByLimit(articleReferences, limit);
   }
+
+  Future<ArticleEntity> getSingleArticle(String articleID) async {
+    ArticleEntity articleEntity = await _firestoreManager.getArticleById(articleID);
+    await _getListOfArticlesWithImages([articleEntity]);
```

Comment:
Probably here we should call  _getArticleWithImage instead of calling the _getListOfArticlesWithImages giving just one Article, is a little bit confusing when reading the code.